### PR TITLE
docs: adds README for node

### PR DESCRIPTION
**Pull request by petethepig**

### CODE REVIEW

**Review by Rperry2174 — APPROVED** [review]

File: examples/language-sdk-instrumentation/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+## Continuous Profiling for Node applications
+### Profiling a Node Rideshare App with Pyroscope
+
+![golang_example_architecture_new_00](https://user-images.githubusercontent.com/23323466/173370161-f8ba5c0a-cacf-4b3b-8d84-dd993019c486.gif)
+
+Note: For documentation on Pyroscope's Node integration visit [our website](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/nodejs/).
+
+## Background
+In this example we show a simplified, basic use case of Pyroscope. We simulate a "ride share" company which has three endpoints found in `main.js`:
```
Comment (knylander-grafana) [inline examples/language-sdk-instrumentation/nodejs/README.md:9]:
```suggestion
In this example, we show a simplified, basic use case of Pyroscope. We simulate a "ride share" company which has three endpoints found in `main.js`:
```

File: examples/language-sdk-instrumentation/nodejs/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+docker pull grafana/pyroscope:latest
+
+# Run the example project:
+docker-compose up --build
+
+# Reset the database (if needed):
+docker-compose down
+```
+
+This example will run all the code mentioned above and also send some mock-load to the 3 servers as well as their respective 3 endpoints. If you select our application: `nodejs.wall` from the dropdown, you should see a flame graph that looks like this (below). After we give 20-30 seconds for the flame graph to update and then click the refresh button we see our 3 functions at the bottom of the flame graph taking CPU resources _proportional to the size_ of their respective `search_radius` parameters.
```
Comment (knylander-grafana) [inline examples/language-sdk-instrumentation/nodejs/README.md:56]:
```suggestion
This example will run all the code mentioned above and also send some mock-load to the 3 servers as well as their respective 3 endpoints. If you select our application: `nodejs.wall` from the dropdown, you should see a flame graph that looks like this (below).
After 20-30 seconds, the flame graph updates. Click the refresh button. We see our 3 functions at the bottom of the flame graph taking CPU resources _proportional to the size_ of their respective `search_radius` parameters.
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the REAME. I've added two minor suggestions.

File: examples/language-sdk-instrumentation/nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Resulting flame graph / performance results from the example
+### Running the example
+
+There are 3 examples:
+* `express` - basic integration example
+* `express-ts` - type script example
+* `express-pull` — pull mode example
+
+To run any of them example run the following commands:
```
Comment (knylander-grafana) [inline examples/language-sdk-instrumentation/nodejs/README.md:41]:
```suggestion
To use any of the examples, run the following commands:
```

**Comment by knylander-grafana** [comment]
@petethepig I did another quick review and found a missing comma. Everything looks good. Okay to merge?

**Pull request merged**